packages/api/app/Controllers/Http/ServiceController.js

```diff
@@ -13,6 +13,7 @@ class ServiceController {
 	async index({ request }) {
 		const filters = request.all();
 		return Service.query()
+			.where({ active: true })
```

Comment:
Put this condition inside the model. These changes will be useful when we need to extend the service available logic.